packages/esm-appointments-app/src/form/appointments-form.component.tsx

```diff
@@ -5,8 +5,6 @@ import { useTranslation } from 'react-i18next';
 import {
   Button,
   ButtonSet,
-  DatePicker,
-  DatePickerInput,
```

Comment:
So, at least some of the tests are failing because these imports were removed but we're still referencing those components in this file.

Reply:
I think it makes sense to go one datepicker at a time, but we should leave these here in the interim.